libvast/builtins/formats/json.cpp

```diff
@@ -265,21 +270,21 @@ class plugin final : public virtual parser_plugin,
     };
   }
 
-  [[nodiscard]] auto default_saver(const record&) const
-    -> std::pair<std::string, record> override {
-    return {"stdout", record{}};
+  auto default_saver(std::span<std::string const> args) const
+    -> std::pair<std::string, std::vector<std::string>> override {
+    (void)args; // TODO
```

Comment:
The problem is that we cannot return an error here. Options: `VAST_ASSERT`, `VAST_WARN` or ignore. What do you think is best?

Reply:
I think this API is rather unfortunate in particular for connectors and formats that actually parse arguments (such as your file loader).

Reply:
I was able to make it work somehow in my PR, but I'm still left confused about the purpose of:
`(void)args; // TODO`

Reply:
I took the option to ignore the arguments (and therefore potential errors) for now here. Your PR also does not do proper argument parsing in the corresponding function and also ignores errors. The `// TODO` is because I was not sure about this behavior. But given that your implementation has a similar outcome, this might be fine (for now).

Reply:
Can you use `[[maybe_unused]]` for the `args` parameter then?

Reply:
I removed the argument name in 7580fc51d9f67349f7474e459f017431146535d2, which does more or less the same.